test/acceptance/cli-test.js

```diff
@@ -351,8 +351,8 @@ describe('ember-template-lint executable', function () {
       });
     });
 
-    describe('given path to single file with custom extension with errors', function () {
-      it('should print errors', async function () {
+    describe('given path to single file with custom extension', function () {
+      it('should not print errors', async function () {
```

Comment:
This isn't a mistake, we do support passing arbitrary files.
Can you revert the changes to this test?